### PR TITLE
Restructure Consulting Page

### PR DESCRIPTION
**Pull request by NPDebs**
This PR relates to PR #680.
After a sucessful merge, it was suggested in this [comment](https://github.com/open-life-science/open-life-science.github.io/pull/699#issuecomment-1799481652) that the page could do without the consulting.yaml file in the
`_data` folder.

As such, in this PR, the following changes were made:
- the `consulting.yaml` file removed,
- a new _consulting_ collection created,
- data previously stored in _consulting.yaml_ were added diretly to the metadata of each file,
- files in the _consulting_ collection were looped through and displayed in the index page.

The page looks like this now:
Screenshot (591)

Thanks for reviewing!

### CODE REVIEW

File: _consulting/accessibility-consulting.md
Comment (bebatut) [inline _consulting/accessibility-consulting.md:1]: Does it need to be `_consulting` instead of `consulting`?

**Review by bebatut — COMMENTED** [review]
Thanks a lot for this PR. It looks great!!

File: _consulting/accessibility-consulting.md
Comment (NPDebs) [reply _consulting/accessibility-consulting.md:1]: Yes, please. This is generally the naming convention for _collections_ in Jekyll.

**Comment by NPDebs** [comment]
> Thanks a lot for this PR. It looks great!!

Thank you! 😍

File: _consulting/accessibility-consulting.md
Comment (bebatut) [reply _consulting/accessibility-consulting.md:1]: True

**Comment by bebatut** [comment]
Thanks @NPDebs

**Pull request merged**